recipes/taocpp-taopq/all/test_package/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.8)
```

Comment:
Just a quick question.
Is there any reason to upgrade the minimum required version of cmake to 3.8?

Reply:
C++17

Reply:
I understand. Thank you.